### DISCUSSION TITLE
Shockley-Read-Hall Recombination

### DISCUSSION THREAD

**Discussion by Samarmf** [materialsproject/pymatgen#4238] [Q&A]
I am trying to replicate the example here https://materialsproject.github.io/pymatgen-analysis-defects/content/nonradiative.html but I cannot find the files, can someone please help?

**Comment by DanielYang59** [comment]
@Samarmf Happy new year and thanks for reaching out.

Note the `pymatgen-analysis-defects` package as a plugin of `pymatgen` has [its own repo](https://github.com/materialsproject/pymatgen-analysis-defects)

If I understand your question correctly that you're asking for the "test files", [should be this](https://github.com/materialsproject/pymatgen-analysis-defects/tree/main/tests/test_files)

**Comment by Samarmf** [comment]
Thanks a lot for your response, these are single point calculations,
correct?




Best Regards,
Samar Fawzy, PhD
The American University in Cairo
